example/package.json

```diff
@@ -45,30 +45,30 @@
     ]
   },
   "dependencies": {
-    "@cumulus/async-operations": "8.1.0-alpha.1",
-    "@cumulus/api": "8.1.0",
-    "@cumulus/api-client": "8.1.0",
-    "@cumulus/aws-client": "8.1.0",
-    "@cumulus/checksum": "8.1.0",
-    "@cumulus/cmr-client": "8.1.0",
-    "@cumulus/cmrjs": "8.1.0",
-    "@cumulus/common": "8.1.0",
-    "@cumulus/discover-granules": "8.1.0",
-    "@cumulus/discover-pdrs": "8.1.0",
-    "@cumulus/files-to-granules": "8.1.0",
-    "@cumulus/hello-world": "8.1.0",
-    "@cumulus/ingest": "8.1.0",
-    "@cumulus/integration-tests": "8.1.0",
-    "@cumulus/message": "8.1.0",
-    "@cumulus/move-granules": "8.1.0",
-    "@cumulus/parse-pdr": "8.1.0",
-    "@cumulus/pdr-status-check": "8.1.0",
-    "@cumulus/post-to-cmr": "8.1.0",
-    "@cumulus/queue-granules": "8.1.0",
-    "@cumulus/queue-pdrs": "8.1.0",
-    "@cumulus/sf-sqs-report": "8.1.0",
-    "@cumulus/sync-granule": "8.1.0",
-    "@cumulus/test-processing": "8.1.0"
+    "@cumulus/async-operations": "9.0.1-alpha.1",
```

Comment:
Should just be `9.0.1` since that package version is released now. The alpha package was for pre-release

Reply:
AH gotcha